src/Appium.Net/Appium/Service/AppiumLocalService.cs

```diff
@@ -58,7 +58,7 @@ public class AppiumLocalService : ICommandServer
         /// <summary>
         /// The base URL for the managed appium server.
         /// </summary>
-        public Uri ServiceUrl => new Uri($"http://{IP.ToString()}:{Convert.ToString(Port)}/wd/hub");
+        public Uri ServiceUrl => new Uri($"http://{IP}:{Convert.ToString(Port)}");
```

Comment:
this will make the service fully incompatible with Appium 1. Also, one could provide --base-path/-bp argument to change this path. We should be able to support such approach. At least Python and Java clients do

Reply:
@mykola-mokhnach , already discussed the fact it will be incompatible with appium 1 with @akinsolb, this is the reason we will merge to release 5.0 branch.
I will try to look what I can do with the --base-path/-bp argument in order to support it fully.

Reply:
I didn't know the other bindings supported v1. It would be ideal if we could support both, where users can build their own client

Reply:
@akinsolb /@mykola-mokhnach Do you prefer I add the base path support as part of the arguments in the AppiumLocalService or it's better to create a dedicated var for it? just like we have IP and Port

Reply:
I don't have any strong pref there. Do it the way which is closer/more straightforward to dotnet client users. In java and python service implementations we  fetch base path value from the corresponding CLI argument. `/` is used by default.

Reply:
I've pushed some changes to this PR so we will support base-path, but I still need to work on the ipv4 status issue.